Vizzuality/heco-invest
# feat: Job API for closing open calls past deadline

This PR adds new API `POST /jobs/open_calls/close_past_deadline` which automatically changes status of open calls to `closed` if `closing_at` is at past.

It also adds callback which ensures that anytime somebody manually closes open call, its future deadline is changed to current time. There is also new validation `closing_at_is_at_future ` which makes sure that `closing_at` is at future.

## Testing instructions

New Job API should be available via Rswag. It should be also possible to check via it that all newly closed open calls have automatically changed deadline to current time.

## Tracking

https://vizzuality.atlassian.net/browse/LET-690
https://vizzuality.atlassian.net/browse/LET-947
